src/RemoteLRS.php

```diff
@@ -484,42 +484,30 @@ public function retrieveVoidedStatement($id, $options = array()) {
     private function _queryStatementsRequestParams($query) {
         $result = array();
 
-        foreach (array('agent') as $k) {
-            if (isset($query[$k])) {
-                $result[$k] = json_encode($query[$k]->asVersion($this->version));
-            }
+        if (isset($query['agent'])) {
+            $result['agent'] = json_encode($query['agent']->asVersion($this->version));
         }
-        foreach (
-            array(
-                'verb',
-                'activity',
-            ) as $k
-        ) {
+
+        $identifiables = ['verb', 'activity'];
+        foreach ($identifiables as $k) {
             if (isset($query[$k])) {
                 $result[$k] = $query[$k]->getId();
             }
         }
-        foreach (
-            array(
-                'ascending',
-                'related_activities',
-                'related_agents',
-                'attachments',
-            ) as $k
-        ) {
+
+        $arguments = ['ascending', 'related_activities', 'related_agents', 'attachments'];
+        foreach ($arguments as $k) {
             if (isset($query[$k])) {
                 $result[$k] = $query[$k] ? 'true' : 'false';
             }
         }
-        foreach (
-            array(
-                'registration',
-                'since',
-                'until',
-                'limit',
-                'format',
-            ) as $k
-        ) {
+
+        $content = [
+            'registration', 'since',
+            'until', 'limit',
+            'format'
+        ];
+        foreach ($content as $k) {
```

Comment:
Why make these separated out, and then change the original array construct for agent to be an inlined version? Does this change apply something for PHP7?